packages/url-launcher/android/src/main/java/io/flutter/plugins/url_launcher/UrlLauncherPlugin.java

```diff
@@ -13,34 +18,51 @@
  * UrlLauncherPlugin
  */
 public class UrlLauncherPlugin implements MethodCallHandler {
-  private FlutterActivity activity;
-
-  public static UrlLauncherPlugin register(FlutterActivity activity) {
-    return new UrlLauncherPlugin(activity);
-  }
-
-  private UrlLauncherPlugin(FlutterActivity activity) {
-    this.activity = activity;
-    new MethodChannel(
-            activity.getFlutterView(), "plugins.flutter.io/URLLauncher").setMethodCallHandler(this);
-  }
-
-  @Override
-  public void onMethodCall(MethodCall call, Result result) {
-    if (call.method.equals("UrlLauncher.launch")) {
-      launchURL((String) call.arguments);
-      result.success(null);
-    } else {
-      result.notImplemented();
+    private FlutterActivity activity;
+
+    public static UrlLauncherPlugin register(FlutterActivity activity) {
+        return new UrlLauncherPlugin(activity);
     }
-  }
-  private void launchURL(String url) {
-    try {
-      Intent launchIntent = new Intent(Intent.ACTION_VIEW);
-      launchIntent.setData(Uri.parse(url));
-      activity.startActivity(launchIntent);
-    } catch (java.lang.Exception exception) {
-      // Ignore parsing or ActivityNotFound errors
+
+    private UrlLauncherPlugin(FlutterActivity activity) {
+        this.activity = activity;
+        new MethodChannel(
+                activity.getFlutterView(), "plugins.flutter.io/URLLauncher").setMethodCallHandler(this);
```

Comment:
Decide on format for the channel name suffix: camelCase, multi_word, acronyms in lower or upper case, whatever. Just be consistent across plugins.